features/fetch-with-ignore-git.feature

```diff
@@ -0,0 +1,102 @@
+Feature: Fetch with ignore in git
+
+    Sometimes you want to ignore files from a project
+    These can be specified using the `ignore:` tag
+
+    Background:
+        Given a git-repository "SomeInterestingProject.git" with the files
+            | path                                        |
+            | SomeFolder/SomeSubFolder/SomeFile.txt       |
+            | SomeFolder/SomeSubFolder/OtherFile.txt      |
+            | SomeFolder/SomeSubFolder/SomeFile.md        |
+            | SomeFolder/SomeOtherSubFolder/SomeFile.txt  |
+            | SomeFolder/SomeOtherSubFolder/OtherFile.txt |
+
+    Scenario: A file pattern is fetched from a repo
+        Given the manifest 'dfetch.yaml' in MyProject
+            """
+            manifest:
+                version: 0.0
+                projects:
+                    - name: SomeInterestingProject
+                      url: some-remote-server/SomeInterestingProject.git
+                      src: SomeFolder/SomeSubFolder
+                      ignore:
+                         - OtherFile.txt
+                      tag: v1
```

Comment:
The repo mentioned doesn't have this tag, but it is fetched?!

Reply:
The generated local git repo always has an implicit v1 tag:

https://github.com/dfetch-org/dfetch/blob/c8f3b2200f5464a66dad41be79248190f191e31e/features/steps/git_steps.py#L92